### ISSUE TITLE
Server Count not Working

### ISSUE THREAD

**Issue by sdfsdfsdf1234** [jellz/discordboats.club#46]
### Expected behaviour?
I post the bot server count, get a success response, and the server count shows up on the bot page.
### Actual behaviour?
I post the bot server count, get a success response, and nothing shows up on the bot page.
### Steps to reproduce?
Post server count
### Any additional information?
I've tried in postman and from my bot in nodejs and the same result, successful response, but nothing on the website.

**Comment by jellz** [comment]
Doesn't use bug format, negado.

**Comment by jellz** [comment]
Would you be able to provide a link to your bot page?

**Comment by sdfsdfsdf1234** [comment]
https://discordboats.club/bot/403341014850469890

**Comment by RONTheCookie** [comment]
alright i changed some stuff in https://github.com/jellz/discordboats.club/commit/ee390872f45628a776406556054252efd16a9d14 , could you try again?

**Comment by sdfsdfsdf1234** [comment]
It still isn't working, my code posts server count whenever it joins/leaves, and it's had activity recently. Other bot sites have correct servers, so I think it's still the site

**Comment by sdfsdfsdf1234** [comment]
let me get a screenshot of my posting code

**Comment by sdfsdfsdf1234** [comment]
http://sdf-images.000webhostapp.com/i/xnozpd.png

**Comment by sdfsdfsdf1234** [comment]
Ignore the fact that the send line is commented, I just have it like that on my laptop because the beta version shouldn't send server count

**Comment by sdfsdfsdf1234** [comment]
On the real instance of the bot, it is all uncommented

**Comment by sdfsdfsdf1234** [comment]
and like I said, it gets a success response, so I'm pretty sure all my stuff is correct. also you asked if it was using the library, it is not

**Comment by sdfsdfsdf1234** [comment]
Oh wait, I misread your question. I tested it with the module on runkit and it did work, but mine is getting a success response so I see no reason why mine doens't

**Comment by sdfsdfsdf1234** [comment]
Does the api require a content type header of application/json? I looked at the code earlier, and unless express or something checks that I didn't see anything about requiring application/json

**Comment by RONTheCookie** [comment]
Yeah, it probably does - I don't know how express works but I'd assume it checks it, Also I just found that it lets the request go through if the body does not exist - and I will fix it.

**Comment by sdfsdfsdf1234** [comment]
Ok, so do I just need to add the header `Content-Type: application/json`?
